app/src/main/java/es/wolfi/app/passman/CustomFieldViewAdapter.java

```diff
@@ -0,0 +1,131 @@
+/**
+ * Passman Android App
+ *
+ * @copyright Copyright (c) 2016, Sander Brand (brantje@gmail.com)
+ * @copyright Copyright (c) 2016, Marcos Zuriaga Miguel (wolfi@wolfi.es)
+ * @license GNU AGPL version 3 or any later version
+ * <p>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * <p>
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package es.wolfi.app.passman;
+
+import android.annotation.SuppressLint;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.recyclerview.widget.RecyclerView;
+
+import org.json.JSONException;
+
+import java.util.List;
+
+import es.wolfi.passman.API.CustomField;
+import es.wolfi.passman.API.File;
+
+/**
+ * {@link RecyclerView.Adapter} that can display a {@link File} and makes a call to the
+ * specified {@link CredentialDisplay.OnListFragmentInteractionListener}.
+ * TODO: Replace the implementation with code for your data type.
+ */
+public class CustomFieldViewAdapter extends RecyclerView.Adapter<CustomFieldViewAdapter.ViewHolder> {
+
+    private final List<CustomField> mValues;
+    private final CredentialDisplay.OnListFragmentInteractionListener customFieldListListener;
+
+    public CustomFieldViewAdapter(List<CustomField> customFields, CredentialDisplay.OnListFragmentInteractionListener listener) {
+        mValues = customFields;
+        customFieldListListener = listener;
+    }
+
+    @Override
+    public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext())
+                .inflate(R.layout.fragment_credential_custom_field_item, parent, false);
+        return new ViewHolder(view);
+    }
+
+    @SuppressLint("DefaultLocale")
+    public static String humanReadableByteCount(long bytes, boolean si) {
+        int unit = si ? 1000 : 1024;
```

Comment:
There is already an implementation of this function [here](https://github.com/nextcloud/passman-android/blob/master/app/src/main/java/es/wolfi/utils/FileUtils.java#L374)

Reply:
fixed in b0f83244b829a59aed74bb30a44134a98c0509d2